python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -1369,7 +1504,11 @@ def min_max_scale(self, columns, min=0.0, max=1.0):
 
         return FeatureTable(df), min_max_dict
 
-    def transform_min_max_scale(self, columns, min_max_dict):
+    def transform_min_max_scale(
+        self,
+        columns: Union[str, List[str]],
+        min_max_dict: Dict[str, Union[Tuple[float, float], Tuple[List[float], List[float]]]]
```

Comment:
the value of dict can only be tuple (float, float)?

Reply:
> the value of dict can only be tuple (float, float)?

i think this is consistent with `transform_min_max_scale`?
```
    def transform_min_max_scale(
        self,
        columns: Union[str, List[str]],
        min_max_dict: Dict[str, Union[Tuple[float, float], Tuple[List[float], List[float]]]]
    ) -> "FeatureTable":
```

Reply:
yes, it's correct